presto-spi/src/main/java/com/facebook/presto/spi/type/VarcharType.java

```diff
@@ -29,6 +29,11 @@
     public static final int MAX_LENGTH = Integer.MAX_VALUE;
     public static final VarcharType VARCHAR = new VarcharType(MAX_LENGTH);
 
+    public static VarcharType createUnboundedVarcharType()
```

Comment:
This seems to be invalid name, this method does not create anything.
Why not just just VARCHAR constant btw?